docs/source/features/sheet_management/index.rst

```diff
@@ -4,3 +4,126 @@
 ##################
  Sheet Management
 ##################
+
+In the Graviti SDK, ``Sheet`` and ``DataFrame`` are interpretations of the same thing at
+different levels.
+
+Sheet refers to the form of data organization that is one level lower than the dataset. One
+dataset can have many different sheets, such as train, test, or frame-by-frame pictures from
+different videos. Each sheet has its own schema.
+
+Graviti SDK organizes the data of a sheet into a DataFrame format, which makes it more convenient
+and intuitive to get and modify the data.
+
+More details about the ``Sheet`` and ``Schema`` are as follows:
+
+.. toctree::
+   :maxdepth: 1
+
+   dataframe
+   schema
+
+The following will introduce the sheet-related operations in the SDK. Of course, it is
+necessary to get a dataset first:
+
+.. code:: python
+
+   from graviti import Workspace
+
+   ws = Workspace(f"{YOUR_ACCESSKEY}")
+   dataset = ws.datasets.get(f"{DATASET_NAME}")
+
+****************
+ Create a Sheet
+****************
+
+Create a sheet on the open draft:
+
+.. code:: python
+
+   from graviti import DataFrame
+
+   draft = dataset.drafts.create(f"{DRAFT_TITLE}")
+   draft[f"{SHEET_NAME}"] = DataFrame(data=data, schema=schema)
+   draft[f"{SHEET_NAME}"] = DataFrame(schema=schema)
+   draft.upload()
+
+.. note::
+   Since only open drafts can upload the changes to sheet, it is better to use draft instead
```

Comment:
`use draft instead of dataset`?
What do you mean by `sheet modification operations`? Aren't only drafts that can modify the data?

Reply:
no, dataset can also modify the data, but it can not upload.